alignak/objects/satellitelink.py

```diff
@@ -241,23 +240,24 @@ def add_failed_check_attempt(self, reason=''):
         if self.attempt == self.max_check_attempts:
             self.set_dead()
 
-    def update_infos(self):
+    def update_infos(self, now):
```

Comment:
Only because we reduce now() calls in dispatcher.check_alive() because can have many satellites.
Seems better use it like this no?